elixir/apps/web/lib/web/live/policies/edit.ex

```diff
@@ -1,20 +1,28 @@
 defmodule Web.Policies.Edit do
   use Web, :live_view
 
+  alias Domain.Policies
+
+  def mount(%{"id" => id} = _params, _session, socket) do
+    {:ok, policy} = Policies.fetch_policy_by_id(id, socket.assigns.subject)
+
+    {:ok, assign(socket, policy: policy)}
+  end
+
   def render(assigns) do
     ~H"""
     <.breadcrumbs home_path={~p"/#{@account}/dashboard"}>
       <.breadcrumb path={~p"/#{@account}/policies"}>Policies</.breadcrumb>
-      <.breadcrumb path={~p"/#{@account}/policies/DF43E951-7DFB-4921-8F7F-BF0F8D31FA89"}>
-        Engineering access to GitLab
+      <.breadcrumb path={~p"/#{@account}/policies/#{@policy.id}"}>
```

Comment:
No need to keep the `id`, LiveView will take it from the schema:
```suggestion
      <.breadcrumb path={~p"/#{@account}/policies/#{@policy}"}>
```

Reply:
Ah, ok, didn't realize that. 👍

Reply:
I believe I've found and updated all instances of `@policy.id` in paths